### PR TITLE
Migrate to vue-cli 3

### PR DESCRIPTION
**Pull request by paulpestov**
Changes include: 
 
- Migration to vue-cli 3
- Updates in e2e testing environment
- Legal info
- Cleanings

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: 1.0.{build}
```
Comment (ipf) [inline appveyor.yml]: I think we don't need that. AppVeyor is already integrated and may soon be dropped in favor of Github Actions

File: package.json [hunk 2]
```diff
@@ -2,114 +2,60 @@
...
+    "unit": "vue-cli-service test:unit",
     "update-gh-pages": "git checkout gh-pages && git rebase master && git push -f origin gh-pages && git checkout master",
-    "preversion": "rimraf node_modules && npm install && npm run test",
-    "version": "npm run build && git add dist",
-    "postversion": "git push && git push --tags && npm run update-gh-pages && echo Tag v$npm_package_version created. Run \"npm publish\" to publish \"$npm_package_name\" on npmjs.com."
+    "version": "npm run build && git add dist"
   },
-  "devDependencies": {
-    "autoprefixer": "^9.7.1",
+  "dependencies": {
```
Comment (ipf) [inline package.json]: All dependencies should go to devDependencies as we don't have any runtime deps

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,13 +1,29 @@
-@import 'functions/*';
+@import 'functions/shade.scss';
+@import 'functions/tint.scss';
 
 @import 'util/settings';
 
-@import 'mixins/*';
```
Comment (ipf) [inline src/styles/main.scss]: All these imports may be done with the already-added glob-importer

**Review by ipf — CHANGES_REQUESTED** [review]
Great work - just some small additions

File: appveyor.yml (hunk 1 above)
Comment (paulpestov) [reply appveyor.yml]: done

File: package.json (hunk 2 above)
Comment (paulpestov) [reply package.json]: done

**Comment by ipf** [comment]
:x: [Build tify 1.0.158 failed](https://ci.appveyor.com/project/ipf/tify/builds/34535344) (commit https://github.com/tify-iiif-viewer/tify/commit/3eb24d5dbd by @)

**Comment by ipf** [comment]
:white_check_mark: [Build tify 1.0.159 completed](https://ci.appveyor.com/project/ipf/tify/builds/34537317) (commit https://github.com/tify-iiif-viewer/tify/commit/1da70aba3b by @)

**Comment by ipf** [comment]
:x: [Build tify 1.0.160 failed](https://ci.appveyor.com/project/ipf/tify/builds/34569350) (commit https://github.com/tify-iiif-viewer/tify/commit/6a69489f01 by @)

**Comment by ipf** [comment]
:white_check_mark: [Build tify 1.0.161 completed](https://ci.appveyor.com/project/ipf/tify/builds/34569531) (commit https://github.com/tify-iiif-viewer/tify/commit/30941a06bf by @)

**Review by ipf — APPROVED** [review]
I'm fine with the changes - Ready for merge?

**Comment by ipf** [comment]
:x: [Build tify 1.0.168 failed](https://ci.appveyor.com/project/ipf/tify/builds/34633262) (commit https://github.com/tify-iiif-viewer/tify/commit/526c560e03 by @)

**Comment by ipf** [comment]
:x: [Build tify 1.0.169 failed](https://ci.appveyor.com/project/ipf/tify/builds/34633513) (commit https://github.com/tify-iiif-viewer/tify/commit/52ab39ac70 by @)

**Comment by ipf** [comment]
:white_check_mark: [Build tify 1.0.170 completed](https://ci.appveyor.com/project/ipf/tify/builds/34633661) (commit https://github.com/tify-iiif-viewer/tify/commit/c9742dd997 by @)

File: package.json [hunk 4]
```diff
@@ -16,93 +16,49 @@
   },
   "license": "AGPL-3.0",
   "scripts": {
-    "prepare": "rimraf openseadragon && git clone https://github.com/openseadragon/openseadragon.git && cd openseadragon && git checkout tags/v2.4.1 && cd .. && npm run fix-openseadragon",
+    "build": "vue-cli-service build",
+    "e2e": "node tests/e2e/runner.js chrome",
     "fix-openseadragon": "replace \"\\}\\(this, function \\(\\) \\{\" \"}(window, function () {\" ./openseadragon/src/openseadragon.js",
-    "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
```
Comment (ipf) [inline package.json:21]: The removed command should also be changed in the README - there is stil `npm run dev`

File: package.json [hunk 5]
```diff
@@ -16,93 +16,49 @@
...
-    "node-notifier": "^6.0.0",
-    "node-sass": "^4.13.0",
-    "optimize-css-assets-webpack-plugin": "^3.2.1",
-    "ora": "^4.0.2",
-    "phantomjs-polyfill-find": "0.0.1",
-    "phantomjs-prebuilt": "^2.1.16",
-    "portfinder": "^1.0.25",
-    "postcss-import": "^12.0.1",
-    "postcss-loader": "^3.0.0",
+    "node-sass": "^4.12.0",
```
Comment (ipf) [inline package.json]: What do you think about replacing `node-sass` with just `sass` that has no native dependencies?

**Review by ipf — CHANGES_REQUESTED** [review]
Works from my POV only the Readme has to be changed from `npm run dev` to `npm run serve`
Also the branch should be rebased - that there are no conflicts

**Comment by ipf** [comment]
:white_check_mark: [Build tify 1.0.171 completed](https://ci.appveyor.com/project/ipf/tify/builds/34635937) (commit https://github.com/tify-iiif-viewer/tify/commit/6507dc75f8 by @)

**Comment by ipf** [comment]
:white_check_mark: [Build tify 1.0.172 completed](https://ci.appveyor.com/project/ipf/tify/builds/34636747) (commit https://github.com/tify-iiif-viewer/tify/commit/d691404057 by @)

**Comment by ipf** [comment]
:white_check_mark: [Build tify 1.0.173 completed](https://ci.appveyor.com/project/ipf/tify/builds/34637221) (commit https://github.com/tify-iiif-viewer/tify/commit/3c48130132 by @)

**Review by ipf — APPROVED** [review]

**Pull request merged**